scripts/pre_commit_linter.py

```diff
@@ -2457,6 +2517,7 @@ def _check_constants_declaration(self):
                 parsed_script = self.parsed_js_and_ts_files[filepath]
                 parsed_nodes = parsed_script.body
                 components_to_check = ['constant']
+                angular_constants_set = set()
```

Comment:
Start with a list, use that to check uniqueness. Convert both lists to sets only when you are doing the final comparison.

Reply:
Done!